### ISSUE TITLE
chore: skipped or deferred dependency updates

### ISSUE THREAD

**Issue by DerekRoberts** [bcgov/nr-spar#1568]
Assess and catch up on dependencies that have fallen behind.

https://github.com/bcgov/nr-spar/issues/643

Current list:

```
Ignored or Blocked
These are blocked by an existing closed PR and will not be recreated unless you click a checkbox below.

- chore(deps): update maven all non-major dependencies to v6.6.0.final (org.hibernate.orm.tooling:hibernate-enhance-maven-plugin, org.hibernate:hibernate-spatial)
- fix(deps): update dependency aws-amplify to v6
- fix(deps): update dependency web-vitals to v4
- fix(deps): update tanstack-query monorepo to v5 (major) (@tanstack/react-query, @tanstack/react-query-devtools)
```
